### DISCUSSION TITLE
MaequeeLabel for macOS

### DISCUSSION THREAD

**Discussion by rokach** [cbpowell/MarqueeLabel#273] [Q&A]
Hi,
We will see MarqueeLabel for macOS soon?

Thanks!

**Comment by cbpowell** [comment]
Hey! So I haven't looked into that at all, but it would be cool to have. I'm not even sure what the incompatibilities are at the moment. I guess I'll try to investigate it sometime soon, but I'm certainly open to PRs to add it as well.

**Reply by rokach** [reply]
I don't have the acknowledge to "convert" it to macOS but I know UILabel and NSTextField is different, also NSTextField has a placeholder so MaequeeLabel also need to support placeholder

**Reply by rokach** [reply]
> Hey! So I haven't looked into that at all, but it would be cool to have. I'm not even sure what the incompatibilities are at the moment. I guess I'll try to investigate it sometime soon, but I'm certainly open to PRs to add it as well.

Any news?

**Reply by cbpowell** [reply]
Nope sorry, I haven’t had any time to look into it!

**Reply by rokach** [reply]
> Nope sorry, I haven’t had any time to look into it!

Haha ok I need it to my project hope you will have some time soon.
